src/reformat.c

```diff
@@ -502,7 +502,7 @@ static avifBool avifCreateYUVToRGBLookUpTables(float ** unormFloatTableY, float
     const size_t cpCount = (size_t)1 << depth;
 
     assert(unormFloatTableY);
-    *unormFloatTableY = avifAlloc(cpCount * sizeof(**unormFloatTableY));
+    *unormFloatTableY = (float *)avifAlloc(cpCount * sizeof(**unormFloatTableY));
```

Comment:
Optional: When there is a cast for the return value, I think using the `float` type in `sizeof` is more readable:

```
    *unormFloatTableY = (float *)avifAlloc(cpCount * sizeof(float));
```

Also at line 516.

Reply:
agreed. Done.